src/scripts/main.js

```diff
@@ -1 +1,73 @@
 'use strict';
+
+const firstPromise = new Promise((resolve, reject) => {
+  const handler = () => {
+    resolve();
+    document.removeEventListener('click', resolve);
```

Comment:
Bug: You are removing the event listener with 'resolve' as a handler, but you added the listener with 'handler'. This will not remove the listener as expected.